app.py

```diff
@@ -846,14 +846,15 @@ async def generate_title(conversation_messages):
     messages.append({"role": "user", "content": title_prompt})
 
     try:
-        azure_openai_client = init_openai_client(use_data=False)
```

Comment:
Is there a reason to remove this flag? It will work best without data.

Reply:
the function definition no longer accepts this kwarg. i didn't check the history about when it was removed, but calling `init_openai_client` is throwing an error in the code as it's currently written. either the `use_data` flag should be added back to the function or it needs to be removed as i've suggested here.
https://github.com/microsoft/sample-app-aoai-chatGPT/blob/main/app.py#L98

Reply:
Ah - you're right. Good catch!